embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/telemetry/TelemetryService.kt

```diff
@@ -0,0 +1,15 @@
+package io.embrace.android.embracesdk.telemetry
+
+internal interface TelemetryService {
+
+    /**
+     * Tracks the usage of a public API by name. We only track public APIs that are called when the SDK is initialized.
```

Comment:
nit: it's worth documenting the convention that strings should use underscores & no spaces